advisories/0010.md

```diff
@@ -0,0 +1,129 @@
+#### Ubuntu
+```
+# apt-get install 'docker-ee=5:23.0.9~4-0*' 'docker-ee-cli=5:23.0.9~4-0*' 'docker-ee-rootless-extras=5:23.0.9~4-0*' 'containerd.io=1.6.28~rc.1-2*'
+```
+Check `runc` version:
+```
+# docker version --format '{{range .Server.Components}}{{if eq .Name "runc"}}{{.Version}}{{end}}{{end}}'
+1.1.11-rc1.m1 <<<<<<<<<<<<<<<
+```
+#### RPM package based OSes
+```
+# yum install 'docker-ee-3:23.0.9-4*' 'docker-ee-cli-1:23.0.9-4*' 'docker-ee-rootless-extras-23.0.9-4*' 'containerd.io-1.6.28-4.1.rc.1.1*'
+```
+Check `runc` version:
+```
+# docker version --format '{{range .Server.Components}}{{if eq .Name "runc"}}{{.Version}}{{end}}{{end}}'
+1.1.11-rc1.m1 <<<<<<<<<<<<<<<
+```
+
+For more details please read the [MCR official installation documentation][mcr.docs]
+
+### MCR 20.10.x 
+If you have EOL MCR 20.10.x we `STRONGLY RECOMMEND` to upgrade to the `MCR 23.0.9-1`.
+
+As a temporary solution for environments running `MCR 20.10.12` or newer, the CVE can be resolved by upgrading only the `containerd.io` package to the one shipped with `MCR 23.0.9-1` using the instructions below. This workaround is only possible on `MCR 20.10.12` or newer because this is the first version of MCR 20.10 which consumed containerd 1.6.
+
+`Note` Please be aware that this is an `EXCEPTIONAL` and `TEMPORARY` solution and no further updates are expected.
+#### Ubuntu
+```
+# apt-get install 'containerd.io=1.6.28~rc.1-2*'
+```
+#### RPM package based OSes
+```
+# yum install 'containerd.io-1.6.28-4.1.rc.1.1*'
+```
+
+### Offline deployment
+1. Download packages from [official Mirantis repository][mcr.repos] according your OS and platform
+2. Copy them to offline nodes (using USB Flash for example)
+3. Install these packages on hosts and check `runc` version
+#### Ubuntu
+```
+# dpkg -i docker-ee_23.0.9~4-0*.deb docker-ee-cli_23.0.9~4-0*.deb docker-ee-rootless-extras_23.0.9~4-0*.deb containerd.io_1.6.28~rc.1-2*.deb
+```
+#### RPM package based OSes
+```
+# rpm -i docker-ee-23.0.9-4*.rpm docker-ee-cli-23.0.9-4*.rpm docker-ee-rootless-extras-23.0.9-4*.rpm containerd.io-1.6.28-4.1.rc.1.1*.rpm
+```
+
+### Upgrade MCR using Launchpad 
+If your MKE cluster is deployed with Launchpad and configuration correspond to current cluster configuration you can 
+update MCR w/ Launchpad using the follow steps:
+1. Download the [Launchpad][launchpad] v1.5.4 version or newer
+2. Configure MCR version `23.0.9`
+For example:
+```
+apiVersion: launchpad.mirantis.com/mke/v1.3
+kind: mke
+metadata:
+....
+spec:
+....
+  "mcr":
+    "channel": "stable"
+    "installURLLinux": "https://get.mirantis.com/"
+    "installURLWindows": "https://get.mirantis.com/install.ps1"
+    "repoURL": "https://repos.mirantis.com"
+    "version": "23.0.9"       <<<<<<<<<<<<<<<
+....
+```
+`Note` MCR version must be exactly `23.0.9` (NOT 23.0.9-1)
+3. Run Launchpad apply with `--force-upgrade` flag:
+   - `./launchpad-linux-<arch> apply --force-upgrade` 
```

Comment:
If you look at our public facing docs (https://docs.mirantis.com/mke/3.7/launchpad/lp-get-started.html#install-launchpad) we tell folks to rename the binary to launchpad.  I'd recommend we stay consistent with those docs here in the advisory.

Reply:
Done